CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing to VS Code Debug Extension for GDB
+# Contributing to the CDT GDB Debug Adapter Extension for Visual Studio Code
```

Comment:
BTW I noticed that the next line still refers to Eclipse CDT instead of CDT cloud. Probably should match https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/?tab=readme-ov-file#contributing

Reply:
Good spot! Will change.

Reply:
Fixed